docs/APIs.md

```diff
@@ -1,80 +1,56 @@
 # Constructors
 
-## [vegaTooltip.vega](#vgtooltip)
+## [vegaTooltip](#tooltip)
 
-`vegaTooltip.vega(vgView[, options])`
+`vegaTooltip(view [, options])`
 
 | Parameter       | Type           | Description     |
 | :-------------- |:--------------:| :-------------- |
-| `vgView`        | [Vega View](https://github.com/vega/vega/wiki/Runtime#view-component-api) | The visualization view. |
+| `view`          | [Vega View](https://github.com/vega/vega/wiki/Runtime#view-component-api) | The visualization view. |
```

Comment:
Update View docs link to https://vega.github.io/vega/docs/api/view/

Reply:
Good catch!